jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -787,8 +788,11 @@ bool java_bytecode_languaget::typecheck(
       break;
     case LAZY_METHODS_MODE_EAGER:
       {
+        symbol_table_buildert suffix_aware_table =
+            symbol_table_buildert::build(symbol_table);
+
         journalling_symbol_tablet journalling_symbol_table =
-          journalling_symbol_tablet::wrap(symbol_table);
+          journalling_symbol_tablet::wrap(suffix_aware_table);
```

Comment:
To check: does the journalling table forward `next_suffix` to its wrapped table?